indico/modules/events/abstracts/controllers/display.py

```diff
@@ -76,3 +81,26 @@ def _process(self):
         sorted_abstracts = sorted(self.abstracts, key=attrgetter('friendly_id'))
         pdf = AbstractsToPDF(self.event_new, sorted_abstracts)
         return send_file('my-abstracts.pdf', pdf.generate(), 'application/pdf')
+
+
+class RHSubmitAbstract(RHMyAbstractsBase):
+    def _process(self):
+        abstract_form_class = make_abstract_form(self.event_new)
+        form = abstract_form_class(event=self.event_new)
+        if form.validate_on_submit():
+            data = form.data
+            if isinstance(data['submitted_for_tracks'], Track):
```

Comment:
We do this in the  management RH too. Should we move this logic into `create_abstract`?